tarocfg/config.go

```diff
@@ -234,7 +234,8 @@ type Config struct {
 
 // HashMailCourierCfg is the config for the hashmail proof courier.
 type HashMailCourierCfg struct {
-	Addr string `long:"addr" description:"The full host:port of the hashmail service which is used to deliver proofs"`
+	Addr                string `long:"addr" description:"The full host:port of the hashmail service which is used to deliver proofs"`
+	ApertureTlsCertPath string `long:"aperturetlspath" description:"Path to the Aperture service TLS certificate"`
```

Comment:
The Taro daemon doesn't need to know that the hashmail server is implemented in Aperture. So this can just be `TlsCertPath` and the `long:"tlscertpath"` to match it.